train/train_process.py

```diff
@@ -90,13 +92,16 @@ def run_training_loop(self):
             games_gen_idx += 1
 
             # Train latest model from training data
-            latest_model, _metrics = self._train(latest_model, games_dir)
-            # TODO log metrics into file
-
+            latest_model, iteration_metrics = self._train(latest_model, games_dir)
+            training_metrics.append(iteration_metrics)
+            
             # Compare latest model to the current best, and switch if better
             best_model = self._compare_models(
                 best_model, latest_model, games_dir, games_gen_idx)
             games_gen_idx += 1
+            
+        self._save_metrics(training_metrics)

```

Comment:
I think better is to write to a metrics file and append each iteration.
It seems to me a lot of the runs we don't wait until it finished, and ill be happy to save these iterations metrics as well.
I actually also implemented it in https://github.com/poja/RL/pull/71/commits/487ab8287e01f1a92ef7d22324a1575c7c94aaa2
Tell me which one do you think is better and lets merge one of them.
Either way, the logging change should be on a differetn commit

Reply:
Yes I agree with you. Will fix it when I rebase on master, after we merge your branch